src/service/auth/auth.js

```diff
@@ -57,38 +67,37 @@ const setAuthPermission = async (password, originHost) => {
  * @param {string} params.code
  * @param {string} params.password
  * @param {boolean} params.isSignUp
- * @param {string} params.challengeType
+ * @param {boolean} params.isRemember
  * @return {Promise<any>}
  */
 const authWithCode = async ({
   email,
   code,
   password,
   isSignUp,
-  challengeType = 'otp',
+  isRemember,
 }) => {
   const url = isSignUp
     ? `${identityBaseUrl}/auth/signup`
     : `${identityBaseUrl}/auth/token`;
 
-  const res = await request.post(url, {
-    challengeType,
-    email,
-    code,
-    password,
-  });
-
-  if (!res.success) throw new Error(res.message);
-
-  return res;
+  return withSuccess(
+    request.post(url, {
+      challengeType: 'otp',
+      email,
+      code,
+      password,
+      remember: isRemember,
+    }),
+  );
 };
 
 /**
  * @param {string} idToken
- * @return {Promise<any>}
+ * @return {Promise<*>}
  */
-const authWithGoogle = idToken =>
-  request
+const authWithGoogle = async idToken => {
```

Comment:
What do you think about adding requester from vault with it's "plugins" model to our utilities package and reuse it everywhere? It should be easier that implementing one thing every time

Reply:
I'm about `requester` abstraction

Reply:
not for this task

Reply:
Of course. Please, create task for that